src/disc-golf-events/DiscGolfEventsComponent.tsx

```diff
@@ -396,11 +399,34 @@ const handleImport = async (file: File) => {
         </label>
       </div>
     )}
-      {discGolfEvents.length > 0
+    <Box sx={{ mb: 3 }}>
+      <Typography variant="h4" component="h1" sx={{ textAlign: "center", mb: 2 }}>
+        Disc Golf Events
+      </Typography>
+    <Box sx={{ 
+        display: 'flex', 
+        alignItems: 'center', 
+        gap: 1,
+        justifyContent: 'flex-start'
+      }}>
+        <Typography variant="body2" sx={{ fontWeight: 500 }}>
+          PDGA only
+        </Typography>
+        <Switch
+          checked={showOnlyPDGA}
+          onChange={(e) => setShowOnlyPDGA(e.target.checked)}
+          color="primary"
+        />
+        <Typography variant="body2" sx={{ color: 'text.secondary' }}>
+          {displayedEvents.length} / {sortedDiscGolfEvents.length} events
+        </Typography>
+      </Box>
+    </Box>
+      {displayedEvents.length > 0
         ? (<ReusableTable
-          title="Disc Golf Events"
+          title=" "
```

Comment:
Maybe we can remove at all property title from `<ReusableTable>`. It is more individual now, not generic.